lms/services/assignment.py

```diff
@@ -71,7 +71,12 @@ def upsert_assignment(
         or create a new one if none exist on the DB.
 
         Any existing document_url for this assignment will be overwritten.
+
+        When a new assignment is created values of `extra`
```

Comment:
This is a weird sentence, could use some help with rewording.

Reply:
> If we detect that the new assignment in the LMS has been copied from a historical assignment (perhaps by using the LMS's "course copy" feature) then some of the new assignment's `extra` values might be taken from the historical assignment.